test/test_core.py

```diff
@@ -5953,6 +5953,12 @@ def test_istream(self):
       self.set_setting('LINKABLE', linkable)
       self.do_core_test('test_istream.cpp')
 
+  @also_with_wasmfs
```

Comment:
No need for the `also_with_wasmfs` decorator here, since the test itself already enables WasmFS.